README.md

```diff
@@ -61,6 +61,7 @@ Commands:
     -p, --password=PASSWORD  Password for PKCS12/JCEKS key stores (reads from TTY if missing).
     -m, --pem                Write output as PEM blocks instead of human-readable format.
     -j, --json               Write output as machine-readable JSON format.
+    -l, --leaf               Only display the first certificate. This flag can be paired with --json.
```

Comment:
On the other hand, can we assume that the leaf comes before any intermediates or roots in a cert chain? Since it looks like that is the convention.

Reply:
It's from TLS: https://datatracker.ietf.org/doc/html/rfc8446#section-4.4.2 (this was the same in earlier versions, too). 

> The sender's certificate MUST come in the first
>   CertificateEntry in the list.  Each following certificate SHOULD
>  directly certify the one immediately preceding it.  Because
>  certificate validation requires that trust anchors be distributed
> independently, a certificate that specifies a trust anchor MAY be
> omitted from the chain, provided that supported peers are known to
> possess any omitted certificates.